Server/middlewares/roleChecker.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Middleware for user role checking.
+ * 
+ * The roleChecker function returns middleware that checks whether a user has a specific role (admin or not).
+ * If the user has the required role, it passes the request to the next handler; otherwise, it returns an access denied error.
+ * 
+ * @param {boolean} bool - A boolean value indicating whether the user must be an administrator (true) or not (false).
+ * @returns {function} Middleware
+ * 
+ */
+
+const roleChecker = (bool) =>  
```

Comment:
Better name for the param and it wouldn't have to be explained in comment ;)

Reply:
Done